datafusion/core/src/physical_plan/joins/symmetric_hash_join.rs

```diff
@@ -1339,6 +1370,11 @@ fn combine_two_batches(
 }
 
 impl SymmetricHashJoinStream {
+    fn size(&self) -> usize {
```

Comment:
Should we also include the other fields like the `input_stream` column_indexes, etc? Maybe that is effectively a constant so it is ok to miss it in the calculation

Reply:
Since `HashJoinExec` did not include those features, I have tried to replicate the same behavior in previous commits. However, I have now added the required memory measures. If this implementation is acceptable, we can proceed with it.